astra-db-java/pom.xml

```diff
@@ -53,6 +53,19 @@
 
     </properties>
 
+    <dependencyManagement>
```

Comment:
As per comment, boms are usually included in `dependencyManagement` so that they specify compatible set of versions without adding actual dependencies. Without this, actual dependencies still need version reference. But more importantly, transitive dependencies to Jackson via other dependencies would not get a consistent version without this.